src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,10 +1,18 @@
+Microsoft.CodeAnalysis.CommandLineArguments.GeneratedArtifactsOutputDirectory.get -> string
```

Comment:
Need to make sure that these are going into 16.10, not 16.9.

Reply:
How do i make sure of that?  is there a special process to make sure things don't go into 16.9?